client/templates/userprofile/profile.js

```diff
@@ -6,4 +6,53 @@ Template.profile.helpers({
 		var first = Meteor.user().realname.split(' ');
 		return first[0];
 	}
+});
+
+Template.profile.events({
+	'click .edit-bio': function() {
+		if (Session.get("gearclicked"))
```

Comment:
There are three primary concerns here:
* 'gearclicked' does not accurately describe the purpose of the session variable, 
* We are using camelCase in variable names, 
* We should not pollute the Session namespace, but rather use template level reactive variable instead.